<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileExplorer</class>
 <widget class="QWidget" name="FileExplorer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="vLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLineEdit" name="pathEdit"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="naviLayout" stretch="2,6">
       <item>
        <widget class="QTreeView" name="dirView"/>
       </item>
       <item>
        <widget class="QTableView" name="filesView"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
